jet_tracking/context.py

```diff
@@ -25,9 +95,9 @@ def __init__(self, signals):
                         'i0': 'CXI:JTRK:REQ:I0',
                         'ratio': 'CXI:JTRK:REQ:RATIO',
                         'dropped': 'CXI:JTRK:REQ:DROPPED',
-                        'camera': 'CXI:SC1:INLINE',
-                        'motor': 'CXI:PI1:MMS:01'}
-#        self.CFG_FILE = 'jt_configs/cxi_config.yml'
+                        'camera': 'XCS:GIGE:LJ2',
+                        'motor': 'XCS:LJH:JET:Y'}
+        #self.CFG_FILE = 'jt_configs/cxi_config.yml'
         self.HUTCH = 'cxi'
         self.EXPERIMENT = 'cxix53419'
```

Comment:
There's some messiness going on in this section with respect to default configs vs config files. I'd recommend consolidating all of this into the config file to avoid confusion.